tap_twitter_ads/__init__.py

```diff
@@ -49,8 +56,7 @@ def main():
             # Error codes: https://developer.twitter.com/en/docs/basics/response-codes
             'retry_on_status': [400, 420, 500, 502, 503, 504],
             'retry_on_timeouts': True,
-            'timeout': (5.0, 10.0)}) # Tuple: (connect, read) timeout in seconds
-
+            'timeout': (request_timeout, request_timeout)}) # Tuple: (connect, read) timeout in seconds
```

Comment:
We want to apply same timeout value for connect and read. So can we write like below?
```suggestion
            'timeout': request_timeout}) # Tuple: (connect, read) timeout in seconds
```
Ref: https://2.python-requests.org/en/master/user/advanced/#timeouts

Reply:
Yes, updated the code